rhscl-mariadb-apb/playbooks/provision.yml

```diff
@@ -5,5 +5,7 @@
   roles:
   - role: ansible.kubernetes-modules
     install_python_requirements: no
+  # TODO: switch to ansibleplaybookbundle.asb-modules when account is created
+  - role: cfchase.asb-modules
```

Comment:
@fabianvf perhaps we should discuss a cleaner way to add these modules without asking a user to add these roles at some point